### PR TITLE
Sorted imports

### PR DESCRIPTION
**Pull request by marcelofabri**
Continued from #991

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1066](https://github.com/realm/SwiftLint/issues/1066)
 
+* Add a rule that enforces alphabetical sorting of imports.
```
Comment (jpsim) [inline CHANGELOG.md]: Opt-in

**Comment by marcelofabri** [comment]
I squashed all commits because I thought that was @sberrevoets's idea, but it seems it removed his name from the history. I'm sorry 😰

**Comment by jpsim** [comment]
It's ok. His name will forever be credited in the changelog. The only place that matters :)

**Comment by sberrevoets** [comment]
Believe it or not, I was actually able to not open my laptop for a few days so didn't get a chance to look finish this up. Thanks for taking this over the finish line @marcelofabri!

**Comment by jpsim** [comment]
I applaud your self-restraint

**Pull request merged**